lib/include.js

```diff
@@ -248,6 +251,9 @@ Inclusion.include = function(objects, include, options, cb) {
       newFilter.where[fkName] = {
         inq: foreignKeys,
       };
+      if (!foreignKeys.length) {
```

Comment:
Can we move this short-circuit logic higher, so that we can skip creation of `newFilter` object altogether? I would personally apply this filter on `listOfFKs` before calling `async.each`.

```js

// Optimization: no need to resolve keys that are an empty array
listOfFKs = listOfFKs.filter(function(keys) { return keys.length > 0; });

async.each(listOfFKs, ...);
```